backend/pkg/encryption/generate_token.go

```
@@ -0,0 +1,14 @@
+package encryption
+
+import (
+	"crypto/rand"
+	"encoding/hex"
+)
+
+// GenerateTokenHex generates a random hex string of the specified length (in bytes)
+// The returned string will be twice the length of the input bytes due to hex encoding
+func GenerateTokenHex(length int) string {
+	b := make([]byte, length)
+	rand.Read(b)
```

golangci-lint (1.64.8)

Error return value of `rand.Read` is not checked

(errcheck)